src/main/java/com/starbucks/service/OrderService.java

```diff
@@ -1,20 +1,38 @@
 package com.starbucks.service;
 
-
 import com.starbucks.model.Order;
-import com.starbucks.repositary.OrderRepositary;
+import com.starbucks.repository.OrderRepository;
+
 import org.springframework.stereotype.Service;
 
+import java.util.UUID;
 
 @Service
 public class OrderService {
-    private final OrderRepositary orderRepositary;
 
-    public OrderService(OrderRepositary orderRepositary) {
-        this.orderRepositary = orderRepositary;
+    private final OrderRepository orderRepository;
+
+    public OrderService(OrderRepository orderRepository) {
+        this.orderRepository = orderRepository;
     }
 
     public void saveOrder(Order order) {
-        orderRepositary.saveOrder ( order );
+        orderRepository.save ( order );
+    }
+
+    public String updateOrder(Order order , UUID id) {
+        if (orderRepository.existsById ( id )) {
+            Order orderDB = orderRepository.findById ( id ).get ( );
+            orderDB.setBeverages ( order.getBeverages ( ) );
+            orderDB.setSize ( order.getSize ( ) );
+            orderDB.setSugar ( order.getSugar ( ) );
+            orderRepository.save ( orderDB );
+        }
+        return "Order " + id + " not exist";
```

Comment:
You should look in todo Project, you should use orderRepository.findById and throw 404 NOT FOUND exception with a meaningful message if object wasn't found in BD. Establishing a call to DB takes a lot of time comparatively to the time we use inside the application or in the DB so we need to use as less calls as it's possible (or reasonable). Here it's not critical because it'll be only one object to update not a list and we won't have a lot of connections in on request but the standard is to use findById and throw 404 if it's not found so you should use it. And Return not a String but an Object so the FrontEnd guy or your API user can use this information to show it to the User or do something with it on his side it's a standard as well. REST has a lot of standards you know..